human_aware_rl/imitation/behavior_cloning_tf2_test.py

```diff
@@ -159,9 +160,12 @@ def _clear_pickle():
     suite.addTest(TestBCTraining('test_save_and_load', args.compute_pickle, args.strict))
     suite.addTest(TestBCTraining('test_training', args.compute_pickle, args.strict))
     suite.addTest(TestBCTraining('test_agent_evaluation', args.compute_pickle, args.strict))
-    suite.addTest(TestBCTraining('test_lstm_save_and_load', args.compute_pickle, args.strict))
-    suite.addTest(TestBCTraining('test_lstm_construction', args.compute_pickle, args.strict))
-    suite.addTest(TestBCTraining('test_lstm_training', args.compute_pickle, args.strict))
-    suite.addTest(TestBCTraining('test_lstm_evaluation', args.compute_pickle, args.strict))
+
+    # LSTM tests break on older versions of tensorflow
+    if tf_version.startswith('2.2'):
```

Comment:
Shoot, I could have sworn I got it working with version 2.0.2... Anyways, I still think this should be an argparse flag that defaults to false, to give the user control over which tests to execute instead of changing execution behavior at runtime based on tf version.

Reply:
I believe this difference came from the hotfix before I started working on `human_aware_rl` (because I didn't meddle with this). I think I am not the right person to judge what to change her